lib/dice/src/deviceid_cert_tmpl.rs

```diff
@@ -0,0 +1,97 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// NOTE: This DER blob, offsets & lengths is mostly generated code.
+// TODO: generate cert template DER from ASN.1 & text config
```

Comment:
My first pass at implementing this did generate this stuff at build time. After bike shedding on that code for a while I decided that committing it would be a distraction for a few reasons: The implementation was pretty gnarly on account of shelling out to the openssl ca tools adding an additional dependency that's awkward (openssl-sys?) and slow to build. The ad-hoc parsing of the certs was ugly too and brittle. All of this would be added only to be removed and replaced with something less horrific.

Adding this back is possible but it'll add stuff to the build that I think we don't want (openssl command line tools) and create a lot of churn in the future. I've added the comment you requested here in my latest update to this PR and I'll mark this conversation as resolved. Don't hesitate to re-open the conversation if you disagree or have any other questions.